src/data/bool.lean

```diff
@@ -132,6 +132,10 @@ theorem band_elim_right : ∀ {a b : bool}, a && b → b := dec_trivial
 
 @[simp] theorem bnot_true : bnot tt = ff := rfl
 
+theorem bnot_iff_not : ∀ {b : bool}, !b ↔ ¬b
+| ff := dec_trivial
+| tt := dec_trivial
```

Comment:
```suggestion
theorem bnot_iff_not : ∀ {b : bool}, !b ↔ ¬b := 
dec_trivial
```
I don't think you need the pattern match.

Reply:
Thanks, I got confused by the fact that this doesn't work when you use a parameter instead of a forall.